data/campaigns/The_South_Guard/scenarios/01_Born_to_the_Banner.cfg

```diff
@@ -177,7 +177,7 @@
         name=start
         [message]
             speaker=narrator
-            message= _ "Deoran found himself put to the test early in his command. Loris, the previous commander, had squandered his time in games and diversions, and bandits had overrun the land..."
+            message= _ "Upon arrival, Deoran found the land overrun by bandits and no sign of Sir Loris and the South Guard."
```

Comment:
This needs some sort of tweak to make "the land" and "no sign" siblings (instead of "bandits" and "no sign").